### PR TITLE
[HOPSWORKS-3157] Add indexes for onlinefs logs

### PR DESCRIPTION
**Pull request by gibchikafa**

### CODE REVIEW

**Review by bubriks — APPROVED** [review]

File: providers/start.rb [hunk 1]
```diff
@@ -58,6 +58,50 @@
...
+             },
+             "fgversion":{
+                "type":"integer"
+             },
+             "fgname":{
+                "type":"text"
+             },
+             "projectid":{
+                "type":"integer"
+             }
```
Comment (bubriks) [inline providers/start.rb:99]:
```suggestion
             "host":{
                "type":"text"
             },
             "log_arguments":{
                "type":"object"
             }
```

Comment (bubriks) [reply providers/start.rb:99]: needed after changes in https://github.com/logicalclocks/clusterj-onlinefs/pull/72

File: templates/default/roles.yml.erb [hunk 2]
```diff
@@ -261,6 +269,13 @@ elastic_exporter_hopsworks:
         - read
         - search
         - get
+    - index_patterns:
+      - "onlinefs_${attr.jwt.projid}-*"
```
Comment (o-alex) [inline templates/default/roles.yml.erb:273]: same comment from hopsworks-ee. This adds a new index per project... is this acceptable in cloud (serverless)?

**Review by o-alex — APPROVED** [review]

**Review by SirOibaf — CHANGES_REQUESTED** [review]
Need to fix the mapping to match the new log_argument pattern before merging.

**Comment by gibchikafa** [comment]
New pr is https://github.com/logicalclocks/elasticsearch-chef/pull/105

**Pull request closed without merging**